WalletWasabi/Extensions/RPCClientExtensions.cs

```diff
@@ -82,7 +82,10 @@ public static async Task<AllFeeEstimate> EstimateAllFeeAsync(this IRPCClient rpc
 
 		var minEstimations = GetFeeEstimationsFromMempoolInfo(mempoolInfo);
 		var minEstimationFor260Mb = new FeeRate((decimal)minEstimations.GetValueOrDefault(260 / 4));
-		var sanityFeeRate = FeeRate.Max(minEstimationFor260Mb, mempoolInfo.GetSanityFeeRate());
+		var minSanityFeeRate = FeeRate.Max(minEstimationFor260Mb, mempoolInfo.GetSanityFeeRate());
+		var estimationForTarget2 = minEstimations.GetValueOrDefault(2);
+		var maxEstimationFor3Mb = new FeeRate(estimationForTarget2 > 0 ? (decimal)estimationForTarget2 : 5_000m);
+		var maxSanityFeeRate = maxEstimationFor3Mb;
```

Comment:
I don't understand this, why not just:
```suggestion
		var maxSanityFeeRate = new FeeRate(estimationForTarget2 > 0 ? (decimal)estimationForTarget2 : 5_000m);
```

Reply:
There is no reason.